test/kube_checks.go

```diff
@@ -176,13 +208,19 @@ func DeploymentScaledToZeroFunc() func(d *appsv1.Deployment) (bool, error) {
 // WaitForLogContent waits until logs for given Pod/Container include the given content.
 // If the content is not present within timeout it returns error.
 func WaitForLogContent(ctx context.Context, client *KubeClient, podName, containerName, namespace, content string) error {
-	return wait.PollImmediate(interval, logTimeout, func() (bool, error) {
-		logs, err := client.PodLogs(ctx, podName, containerName, namespace)
+	var logs []byte
+	waitErr := wait.PollImmediate(interval, logTimeout, func() (bool, error) {
+		var err error
+		logs, err = client.PodLogs(ctx, podName, containerName, namespace)
 		if err != nil {
 			return true, err
 		}
 		return strings.Contains(string(logs), content), nil
 	})
+	if waitErr != nil {
+		return fmt.Errorf("logs do not contain the desired content %q, got %q: %w", content, logs, waitErr)
```

Comment:
```suggestion
		return fmt.Errorf("logs do not contain the desired content %q, got %q: %w", content, string(logs), waitErr)
```

I believe this gets us a messy byte output otherwise.

Reply:
I checked that before, so the [rules](https://golang.org/pkg/fmt/ ) say for byte slices with` %q`: `%q	a double-quoted string safely escaped with Go syntax`. Check [here](https://play.golang.org/p/S1BPRIWTEKB).